frontend/src/pages/modelServing/screens/global/EmptyModelServing.tsx

```diff
@@ -25,8 +27,8 @@ const EmptyModelServing: React.FC = () => {
     return (
       <EmptyDetailsView
         title="No deployed models yet"
```

Comment:
Can you align the titles so they are consistent. One ends with `yet` while the other doesn't.

Reply:
Sure.